### DISCUSSION TITLE
Add refresh/rehydrate button to the "edit item" page

### DISCUSSION THREAD

**Discussion by colemahatmccreary** [directus/directus#15289] [Feature Requests]
Useful in different scenarios, such as with manually triggered flows.

If a flow causes a synchronous update to the triggering record, the user likely needs to see the latest data immediately following execution.

Seems the only option currently is to navigate back/fwd or refresh the entire app after a flow is executed?

There could also be a flag on manual flow triggers stating whether the item should be rehydrated following execution. Either way there'd need to be a 'confirm refresh' pop up if there are unconfirmed edits.

**Comment by azrikahar** [comment]
This isn't exactly a refresh button, but the original use case of "refreshing after manual flow" should be what's currently tracked in #14630 👍

**Reply by colemahatmccreary** [reply]
Sick. Will keep an eye on that. Are you working a lot with flows-related issues/improvements @azrikahar?

**Reply by azrikahar** [reply]
> Are you working a lot with flows-related issues/improvements

It's not exactly something I'm intentionally/actively focused on per se, but resolving issues and/or ironing out certain interactions via improvements in general does mean there can be overlaps with Flows every now and then 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
